src/pages/LandingPage/Charts/VulnerableAreasMap.tsx

```
@@ -0,0 +1,416 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import DeckGL from '@deck.gl/react';
+import { ColumnLayer, GeoJsonLayer, ScatterplotLayer } from '@deck.gl/layers';
+import Map from 'react-map-gl';
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)